src/lang_utils/error_codes/M0197.md

```diff
@@ -0,0 +1,8 @@
+# M0195
```

Comment:
move this file to `src/lang_utils/error_codes/M0195.md`

Reply:
I think the title is just wrong and should be #M0197

Reply:
I added a new error for this.

Reply:
I included the `.md` in the `src/lang_utils/error_codes.ml` (75ac82be4)